rules/detekt/src/main/resources/config/config.yml

```diff
@@ -11,7 +11,7 @@ Compose:
     active: true
   ContentEmitterReturningValues:
     active: true
-  ContentSlotReusedCheck:
+  ContentSlotReused:
```

Comment:
Woops, this rule wasn't properly configured -- which means it wasn't enabled by default, and it should have been. The test caught it, and now it's fixed.